src/frontend/src/test-e2e/flows.ts

```diff
@@ -92,12 +92,8 @@ export const FLOWS = {
     await welcomeView.login();
     await welcomeView.typeUserNumber(userNumber);
     await browser.$("button[data-action='continue']").click();
-    // NOTE: depending on the browser, we issue different warnings. On Safari,
-    // the warning comes before the recovery method selector. Since we only
-    // test on Chrome we always expect the recovery selector first.
-    const recoveryMethodSelectorView = new RecoveryMethodSelectorView(browser);
-    await recoveryMethodSelectorView.waitForDisplay();
-    await recoveryMethodSelectorView.skipRecovery();
+    // NOTE: handle recovery nag because there is no recovery phrase
```

Comment:
Is this a TODO or a note?

Reply:
this is a note.

Reply:
Sorry, I don't understand the note then.

What is it for?

Reply:
See https://github.com/dfinity/internet-identity/pull/1928.